src/main/groovy/com/adarshr/gradle/testlogger/TestLoggerExtension.groovy

```diff
@@ -227,6 +229,15 @@ class TestLoggerExtension extends TestLoggerExtensionProperties {
             .getOrElse(false)
     }
 
+    Boolean getshowOnlySlow() {
```

Comment:
Typo in the method name :) This is why the command line argument wasn't working. It should be `getShowOnlySlow` (capitalised).

Reply:
My mistake. Done ;)